roles/core/pxe_stack/files/osdeploy/redhat_8.ipxe

```diff
@@ -17,11 +17,11 @@ echo | > Target kickstart: ${eq-equipment-profile}.kickstart.cfg
 echo |
```

Comment:
Need to split line 15 above to echo both image and repository roots, same for centos_8, 7, etc.

Reply:
I like verbosity.

Reply:
Done